tests/Middleware/WebActionsCallerTest.php

```diff
@@ -0,0 +1,71 @@
+<?php
+
+namespace Yiisoft\Yii\Web\Middleware;
+
+use Nyholm\Psr7\Response;
+use PHPUnit\Framework\TestCase;
+use Psr\Container\ContainerInterface;
+use Psr\Http\Message\ResponseInterface;
+use Psr\Http\Message\ServerRequestInterface;
+use Psr\Http\Server\RequestHandlerInterface;
+
+class WebActionsCallerTest extends TestCase
```

Comment:
Same as in case with `ActionCallerTest` it worth checking that:

1. Request is passed to `example()`
2. Dependency injection by method signature works in `example()`